grpc/proto/space.proto

```diff
@@ -199,9 +206,9 @@ service SpaceApi {
   }
 
   // Share bucket via public key using Textile Hub inboxing
-  rpc ShareBucketViaPublicKey(ShareBucketViaPublicKeyRequest) returns (ShareBucketViaPublicKeyResponse) {
+  rpc ShareFilesViaPublicKey(ShareFilesViaPublicKeyRequest) returns (ShareFilesViaPublicKeyResponse) {
```

Comment:
Changed this instead of adding a new one since we wont be sharing buckets as a whole anytime soon (unless I'm missing something).